cli/src/test/java/com/devonfw/tools/ide/commandlet/StatusCommandletTest.java

```diff
@@ -0,0 +1,30 @@
+package com.devonfw.tools.ide.commandlet;
+
+import org.junit.jupiter.api.Test;
+
+import com.devonfw.tools.ide.cli.CliArguments;
+import com.devonfw.tools.ide.context.AbstractIdeContextTest;
+import com.devonfw.tools.ide.context.IdeTestContext;
+
+/**
+ * Test of {@link StatusCommandlet}.
+ */
+
+public class StatusCommandletTest extends AbstractIdeContextTest {
+
+  private static final String PROJECT_BASIC = "basic";
+
+  @Test
+  public void testStatusOutsideOfHome() {
+    //arrange
+    IdeTestContext context = new IdeTestContext();
+    CliArguments args = new CliArguments("status");
+    args.next();
+
+    //act
+    context.run(args);
+
+    //assert
+    assertThat(context).logAtWarning().hasMessageContaining("You are not inside an IDE installation: C:");
```

Comment:
This test may work on your windows machine but it may already fail on mine if I am on `D:` and it will fail on every Linux or Mac that does not have a concept of drive letters.